src/OrbitGl/StringManager.cpp

```diff
@@ -10,31 +10,31 @@
 
 #include "OrbitBase/Logging.h"
 
+namespace orbit_gl {
+
 // TODO(b/181207737): Make this assert that it is not present and rename to "Add".
 bool StringManager::AddIfNotPresent(uint64_t key, std::string_view str) {
   absl::MutexLock lock{&mutex_};
-  if (key_to_string_.contains(key)) {
-    ERROR("String collision for key: %ul and string: %s.", key, str);
-    return false;
+  bool inserted = key_to_string_.emplace(key, str).second;
```

Comment:
this changes the logic - we now replace the string in the case of collision. try_emplace maybe?

Reply:
ok, apparently emplace does not replace the element if it exists, I assumed it does.

Reply:
This doesn't change the logic.

`emplace` and `try_emplace` do almost the same thing. The difference among those is something else and is more subtle, the high level semantics is the same.